content/en/apps/tutorials/local-setup.md

```diff
@@ -67,12 +67,15 @@ COUCHDB_PASSWORD=password
 EOF
 ```
 
-Run the following command to start your CHT instance using Docker Compose:
+Run the following command to start your CHT instance using Docker Compose. Ensure that Docker is installed and running:
```

Comment:
The prior section is title "Required Resources" and states:

> Before you begin, you need to have some useful software and tools that are required for things to work:

And then lists docker and docker composer.  This seems redundant and I think we should remove it.

Reply:
I agree